### PR TITLE
Adding Datatables to insurance page

### LINKED ISSUE #6082: slow Insurance company load times Admin->Practice->Practice Settings
### Describe the bug

[forum report](https://community.open-emr.org/t/pagination-on-insurance-companies-list/19789)

### Expected behavior
Insurance companies load quickly

### PR DESCRIPTION
**Pull request by juggernautsei**
Fixes #6082

#### Short description of what this resolves:
After the page loads allow the data to be searchable.

### CODE REVIEW

**Comment by stephenwaite** [comment]
def helpful @juggernautsei but would start a separate issue for it

File: templates/insurance_companies/general_list.html [hunk 1]
```diff
@@ -39,4 +40,12 @@
           {/foreach}
       </tbody>
   </table>
+    <script>
+        {literal}
```
Comment (stephenwaite) [inline templates/insurance_companies/general_list.html:44]: I think the new Smarty() doesn't need the literals any more

**Comment by robertdown** [comment]
Bringing this in, but won't close the attached issue

**Comment by bradymiller** [comment]
2DV

**Pull request merged**